task_handler_layer/task_buoy/msg/ResizedImage.msg

```diff
@@ -0,0 +1,3 @@
+uint32 original_height
```

Comment:
I have already removed that resize_package.xmpl file and the time skrew problem was solved yesterday only that's why I am requesting you for a wiki on removing ROS Lint errors so that I can push it to Main repository.